src/applications/personalization/dashboard-2/tests/unit/healthcare-spinner.unit.spec.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { expect } from 'chai';
+import { renderInReduxProvider } from '~/platform/testing/unit/react-testing-library-helpers';
+import reducers from '~/applications/personalization/dashboard/reducers';
+import HealthCare from '~/applications/personalization/dashboard-2/components/health-care/HealthCare';
+
+describe('HealthCare component', () => {
+  let view;
+  let initialState;
+
+  context('when appointments and messaging data are still loading', () => {
+    it('should only show a loading spinner', async () => {
+      initialState = {
+        user: {
+          profile: {
+            services: ['messaging'],
+          },
+        },
+        health: {
+          appointments: {
+            fetching: true,
+          },
+          msg: {
+            folders: {
+              data: {
+                currentItem: {
+                  loading: true,
+                },
+              },
+              ui: {
+                nav: {
+                  foldersExpanded: false,
+                  visible: false,
+                },
+              },
+            },
+          },
+        },
+      };
+
+      view = renderInReduxProvider(<HealthCare dataLoadingDisabled />, {
```

Comment:
I think you could drop the `dataLoadingDisabled` here. Then the component will make the API calls. It'll fail quickly since the APIs aren't mocked, but we'll still be able to make sure the `progress` bar exists.

Reply:
Good point!